src/org/mozilla/javascript/AbstractEcmaObjectOperations.java

```diff
@@ -24,6 +24,32 @@
         SEALED
     }
 
+    /**
+     * Implementation of Abstract Object operation HasOwnProperty as defined by EcmaScript
+     *
+     * @param cx
+     * @param o
+     * @param pn
+     * @return boolean
+     * @see <a href="https://tc39.es/proposal-accessible-object-hasownproperty/"></a>
+     */
+    static boolean hasPropertyInObject(Context cx, Object o, Object pn) {
```

Comment:
I want the method name to be `hasOwnProperty`.

Reply:
Thank you for pointing out! I refactor in https://github.com/mozilla/rhino/pull/1157/commits/de372ec5f245dbc205351d11212032d5b7f30ee6